addons/sms/models/sms_sms.py

```diff
@@ -39,7 +39,7 @@ class SmsSms(models.Model):
 
     uuid = fields.Char('UUID', copy=False, readonly=True, default=lambda self: uuid4().hex,
                        help='Alternate way to identify a SMS record, used for delivery reports')
-    number = fields.Char('Number')
+    number = fields.Char('Number', compute='_compute_number', readonly=False)
```

Comment:
Oh, you definitely want this to be stored by the way, I don't think it's implicit from `compute` and `readonly=False`

Reply:
@reth-odoo why storing this is necessary?

Reply:
@datonefaridze because it's already stored, and because the sms number doesn't necessarily have to match the number on the partner, there doesn't even need to be a partner.